packages/components/src/input-control/index.js

```diff
@@ -37,6 +37,7 @@ export function InputControl(
 		isPressEnterToChange = false,
 		isFloatingLabel = false,
 		label,
+		labelPosition = 'top',
```

Comment:
Do you see this as something available in all *Control components?

Reply:
Unsure yet! However, I know there are certain UIs where a top vs bottom label position would be beneficial. ColorPicker is a good example. Also BoxControl. I guess we'll see :)